rgw/v2/tests/s3_swift/user_create.py

```diff
@@ -45,6 +46,9 @@ def test_exec(config):
                     json.dump(all_users_info, fout)
                 test_info.success_status("tenanted users creation completed")
 
+        # check sync status if a multisite cluster
+        reusable.check_sync_status()
```

Comment:
@mkasturi18 Could you please add one separate check for metadata sync in https://github.com/red-hat-storage/ceph-qe-scripts/blob/876fa9efae0881c99bd5b8fff5e58f983cd796a4/rgw/v2/lib/sync_status.py

Reply:
@udaysk23 , sure separate check added for metadata sync